src/rpc/api.ts

```diff
@@ -448,4 +448,17 @@ export namespace Api {
     transactionCount: string; // uint32
     ledgerCount: number; // uint32
   }
+
+  export interface BalanceResponse {
+    latestLedger: number;
+    // present only on success, otherwise request malformed or no balance
+    balanceEntry?: {
+      amount: string;
```

Comment:
could `amount` be safely typed more strongly to a 'number' or 'bigint'? maybe there's context for needing string format which I'm not aware. the `amount` might need a brief doc line to state the encoded format so clients can parse correctly?

Reply:
it's because it's a 64-bit int so it gets returned as a string since that doesn't fit for JS. It could be bigint but I don't want to set a new precedent because bigint isn't always supported downstream and we generally use strings for large numbers (e.g. fee stats). I added a comment in 00f77d98 to clarify!